leptonai/cloudrun/remote.py

```diff
@@ -48,20 +38,41 @@ class Remote(object):
     """
 
     _MAX_WAIT_TIME = 600  # In the Remote class, we wait for at most 10 minutes for the photon to be ready.
-    _DEFAULT_TIMEOUT = 600  # In the Remote class, we set the timeout for the deployments to be 10 minutes by default.
+    _DEFAULT_TIMEOUT = 30  # In the Remote class, we set the timeout for the deployments to be 10 minutes by default.
```

Comment:
comment needs to be updated?

Reply:
Yeah, sorry, temporary testing config - will revert back to 600.